raco/myrial/query_tests.py

```diff
@@ -618,6 +618,28 @@ def test_orderby_without_limit_assert(self):
         with self.assertRaises(Exception):  # noqa
             self.check_result(query, None)
 
+    def test_limit_orderby(self):
```

Comment:
Both the bagcomp and SQL ORDER BY syntaxes should have multikey (with varying sort orders) tests.